src/serac/physics/benchmarks/CMakeLists.txt

```diff
@@ -13,20 +13,24 @@ set(physics_benchmark_targets
     )
 
 # Create executable for each benchmark
-foreach(physics_benchmark ${physics_benchmark_targets})
-    blt_add_executable(NAME ${physics_benchmark}
-                       SOURCES ${physics_benchmark}.cpp
+foreach(physics_benchmark_exec ${physics_benchmark_targets})
+    blt_add_executable(NAME ${physics_benchmark_exec}
+                       SOURCES ${physics_benchmark_exec}.cpp
                        DEPENDS_ON ${physics_benchmark_depends}
                        OUTPUT_DIR ${PROJECT_BINARY_DIR}/benchmarks
                        FOLDER serac/benchmarks
                        )
 
     # Add benchmarks with various task counts
     foreach(task_count 1 4 16)
-        blt_add_benchmark(NAME ${physics_benchmark}_${task_count}_task_count
-                          COMMAND ${physics_benchmark}
+        set(physics_benchmark ${physics_benchmark_exec}_${task_count}_task_count)
+        blt_add_benchmark(NAME ${physics_benchmark}
```

Comment:
This ends up being empty due to only changing part of the variables. i would recommend just putting it back to `physics_benchmark`